src/Microsoft.DotNet.Interactive.Tests/InputsWithinMagicCommandsTests.cs

```diff
@@ -237,6 +237,46 @@ public async Task Additional_properties_of_input_request_are_set_by_input_proper
         requestInput.InputTypeHint.Should().Be("file");
     }
 
+    [Theory]
+    [MemberData(nameof(LanguageServiceCommands))]
+    public async Task Language_service_commands_do_not_trigger_input_requests(KernelCommand command)
+    {
+        using var kernel = new CSharpKernel().UseValueSharing();
+
+        bool requestInputWasSent = false;
+
+        kernel.RegisterCommandHandler<RequestInput>((input, _) =>
+        {
+            requestInputWasSent = true;
+
+            return Task.CompletedTask;
+        });
+
+        var result = await kernel.SendAsync(command);
+
+        result.Events.Should().NotContainErrors();
+
+        requestInputWasSent.Should().BeFalse();
```

Comment:
consider: Add an AssertionScope since there are multiple asserts

Reply:
Typically that's only useful if the multiple asserts can fail independently. The first assert here is more of a sanity check.